Common/triggerSel.py

```diff
@@ -0,0 +1,68 @@
+import yaml
+class Triggers():
+    dict_legtypes = {"Electron":"Leg::e", "Muon":"Leg::mu", "Tau":"Leg::tau"}
+
+    def __init__(self, triggerFile):
+        self.trigger_dict = None
+
+        with open(triggerFile, "r") as stream:
+            try:
+                self.trigger_dict= yaml.safe_load(stream)
+            except yaml.YAMLError as exc:
```

Comment:
the execution shouldn't continue if yaml parsing has failed. Remove try/except.

Reply:
ok